components/TextEditor/TextEditorContainer.tsx

```diff
@@ -37,7 +37,7 @@ const TextEditorContainer: FunctionComponent<TextEditorContainerProps> = ({heigh
             />
         </div>
             <Link href={"/submitted"}>
-                <a className={scss.buttonBlack} onClick={handleShowValue}>Submit Code</a>
+                <p className={scss.buttonBlack} onClick={handleShowValue}>Submit Code</p>
```

Comment:
why have we changed this to a `p`?

If it 'does something' it should be a button.
If it 'takes you somewhere' then it should be a link.

If its a link, then every Next `Link` element should always contain an `a` tag.